xbmc/epg/EpgSearchFilter.h

```diff
@@ -66,7 +131,7 @@ namespace EPG
     CDateTime     m_startDateTime;            /*!< The minimum start time for an entry */
     CDateTime     m_endDateTime;              /*!< The maximum end time for an entry */
     bool          m_bIncludeUnknownGenres;    /*!< Include unknown genres or not */
-    bool          m_bPreventRepeats;          /*!< True to remove repeating events, false if not */
+    bool          m_bRemoveDuplicates;          /*!< True to remove duplicate events, false if not */
```

Comment:
Maybe, but in that case the comments should be blocks above the line, not on the same line like this